ambry-api/src/main/java/com/github/ambry/store/StoreGetOptions.java

```diff
@@ -25,8 +25,12 @@ public enum StoreGetOptions {
   Store_Include_Expired,
   /**
    * This option indicates that the store needs to return the message even if it has been
-   * marked for deletion as long as the message has not been physically deleted from the
-   * store.
+   * marked for deletion as long as the message is still in delete retention window
    */
-  Store_Include_Deleted
+  Store_Include_Deleted,
+  /**
+   * This option indicates that the store needs to return the message as long as the message
+   * has not been physically deleted from the store.
+   */
+  Store_Include_Compaction_Ready
```

Comment:
QQ: currently we have several user who enable Store_Include_Deleted to get the blob as long as it not been physically deleted. So seems like this is not a backward compatible change. Will this affect any of their current use cases?